### PR TITLE
Code completion

### PR DESCRIPTION
**Pull request by greatsharma**
This PR adds value of index (2,3) of rowWiseInput matrix at line 230 of MedianImputationTest in imputation_test.cpp.

### CODE REVIEW

File: src/mlpack/tests/imputation_test.cpp [hunk 1]
```diff
@@ -227,6 +227,7 @@ BOOST_AUTO_TEST_CASE(MedianImputationTest)
   BOOST_REQUIRE_CLOSE(rowWiseInput(2, 0), 9.0, 1e-5);
   BOOST_REQUIRE_CLOSE(rowWiseInput(2, 1), 8.0, 1e-5);
   BOOST_REQUIRE_CLOSE(rowWiseInput(2, 2), 4.0, 1e-5);
+  BOOST_REQUIRE_CLOSE(rowWiseInput(2, 3), 8.0, 1e-5);
```
Comment (zoq) [inline src/mlpack/tests/imputation_test.cpp]: Nice catch!

**Comment by ShikharJ** [comment]
Thanks @greatsharma for the contribution.

**Comment by greatsharma** [comment]
> 
> 
> Thanks @greatsharma for the contribution.

Thanks @ShikharJ can you please review my PR #1790, I tested it and also written tests in that PR.

File: src/mlpack/tests/imputation_test.cpp (hunk 1 above)
Comment (greatsharma) [reply src/mlpack/tests/imputation_test.cpp]: Thanks @zoq  can you please review my PR #1790, I tested it and also written tests in that PR.

**Pull request merged**